crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1928,6 +1928,26 @@ impl Client {
         self.send(request, None).await
     }
 
+    /// Change the display name of a device owned by the current user.
+    ///
+    /// Returns a `update_device::Response` which specifies the result
+    /// of the operation.
+    ///
+    /// # Arguments
+    ///
+    /// * `device` - The `Device` to change the display name of.
```

Comment:
`device` should rather be `device_id`, in the rust-sdk a `Device` is a higher level object, not just the identifier: https://matrix-org.github.io/matrix-rust-sdk/matrix_sdk/encryption/identities/struct.Device.html

We're giving the ID of the device we're changing the display name of, not the `Device.